src/components/MyAccount/RequestsTab/CancelButton.tsx

```diff
@@ -10,6 +10,13 @@ import {
 } from "@nypl/design-system-react-components"
 import { BASE_URL } from "../../../config/constants"
 import styles from "../../../../styles/components/MyAccount.module.scss"
+import type {
+  BaseModalProps,
+  BaseProps,
+  ConfirmationModalProps,
+  DefaultModalProps,
+} from "@nypl/design-system-react-components/dist/src/components/Modal/Modal"
```

Comment:
You should now be able to import these types directly from `"@nypl/design-system-react-components"`

Reply:
`BaseProps` is not used in this file so you can remove that import.